app/src/main/java/org/dhis2/usescases/teiDashboard/DashboardRepositoryImpl.java

```diff
@@ -259,6 +249,14 @@ public void setDefaultCatOptCombToEvent(String eventUid) {
                                             ValueUtils.transform(d2, attributeValue, attribute.valueType(), attribute.optionSet() != null ? attribute.optionSet().uid() : null)
                                     );
                                 }
+                            } else {
+                                attributeValues.add(
+                                        TrackedEntityAttributeValue.builder()
+                                                .trackedEntityAttribute(programAttribute.trackedEntityAttribute().uid())
+                                                .trackedEntityInstance(teiUid)
+                                                .value("")
+                                                .build()
+                                );
```

Comment:
do you think you could add some unit test for this method getTEIAttributeValues? Thanks !

Reply:
done!